ext_emconf.php

```diff
@@ -11,9 +11,9 @@
 	'author_email' => '',
 	'constraints' => array(
 		'depends' => array(
-			'typo3' => '10.0.0-10.4.99',
+			'typo3' => '10.0.0-11.0.99',
```

Comment:
can you set this to 11.1.9 or ideally 11.5.99 directly?

same below

Reply:
done